velox/exec/Task.h

```diff
@@ -650,10 +650,6 @@ class Task : public std::enable_shared_from_this<Task> {
   // kFinished.
   bool partitionedOutputConsumed_ = false;
 
-  /// Exchange clients. One per pipeline / source.
-  /// Null for pipelines, which don't need it.
-  std::vector<std::shared_ptr<ExchangeClient>> exchangeClients_;
```

Comment:
Should we move exchangeClientByPlanNode_ also to be next to exchangeClients_? It is a bit strange to have these far away.